index.js

```diff
@@ -126,39 +127,24 @@ module.exports = {
 
     if (existsSync(projectPath)) {
       var babelOptions = getBabelOptions(project);
-      var babelTree = new Babel(this.treeGenerator(projectPath), babelOptions);
+      var babelTree = new Babel(project.treeGenerator(projectPath), babelOptions);
```

Comment:
`treeGenerator` is not a property on the app itself, so this errors when it runs against the actual project itself (since we are doing `[this].concat(plugins, this.project)`